File: exo3/exo3.php
<?php
// count letter
// by fflachet
// beweb

$result = 0;
$i = 0;

function countLetter($word,$letter)
{
	$result = 0;
	$i = 0;
	$argv[1] = $word;
	$argv[2] = $letter;

	echo "$word\n";

	while($i < strlen($word))
	{
		if($word[$i] == $letter)
		{
			$result++;
		}
	$i++;
	}
	echo "le nombre de $letter est : $result\n";
}

countLetter($argv[1], $argv[2])
?>

File: exo3/exo3.java
//count letter
//by fflachet
//beweb

public class exo3
{
	public static void main(String[] args)
	{
		int count = 0;
		String word = args[0];
		String letter = args[1];

		for (int i=0; i < word.length(); i++)
	    {
	        if(word.charAt(i) == letter.charAt(0))
	        {
	    		count++;
	    	}
	    }
	System.out.println("le nombre de "+letter+" est :"+ count);
	}
}